defines.h

```diff
@@ -243,6 +257,8 @@
 /* 4 - x (absolute)   */
 
 #define OP_SIZE_MAX 12
+#define ARCH_STR "MC8600"
```

Comment:
I guess this should be MC6800

Reply:
Yep. But I also just noticed (together with the other ones) that [there is no prefix for 680x and 8008 in the original code](https://github.com/vhelin/wla-dx/pull/184/files#diff-2045016cb90d1e65d71c2407a2570927L164-L199). Should I add the prefix or not? (Same for others)

Reply:
Good point. After thinking about this a little more I think that GB-Z80, Z80, SPC700 and HuC6280 need the "prefixes" (pre-letters?), but others can be just numbers. I guess that's how they are called usually.

Reply:
Well it's just used to display the "help" text, and I think the others can also benefit from the prefix (something that is before of something) because it is then just more explicit.